packages/concerto-cto/src/external.ts

```diff
@@ -0,0 +1,100 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import debug from 'debug';
+import * as pathBrowserify from 'path-browserify';
+import { DefaultFileLoader, FileDownloader } from '@accordproject/concerto-util';
+import { MetaModelUtil, MetaModelNamespace } from '@accordproject/concerto-metamodel';
+
+import Parser = require('./parser');
+
+const debugLog = debug('concerto:ModelManager');
+
+interface Model {
```

Comment:
These interfaces are already defined in the concerto-types package. Please import those instead.

Reply:
Can these interfaces be removed now?